client/packages/invoices/src/InboundShipment/ListView/LinkInternalOrderModal.tsx

```diff
@@ -11,27 +11,29 @@ import {
   DialogButton,
   Typography,
 } from '@openmsupply-client/common';
+import { NameRowFragment } from 'packages/system/src';
 
 interface LinkInternalOrderModalProps {
   isOpen: boolean;
   onClose: () => void;
-  requestRequisitions?: LinkedRequestRowFragment[];
   onRowClick: (row: LinkedRequestRowFragment) => void;
-  isLoading: boolean;
   onNextClick: () => void;
+  name: NameRowFragment | null;
 }
 
 export const LinkInternalOrderModal = ({
   isOpen,
   onClose,
-  requestRequisitions: data,
   onRowClick,
-  isLoading,
   onNextClick: createInvoice,
+  name,
 }: LinkInternalOrderModalProps) => {
   const t = useTranslation();
   const { width, height } = useWindowDimensions();
   const { Modal } = useDialog({ isOpen, onClose });
+  const { data, isLoading } = useInbound.document.listInternalOrders(
+    name?.id ?? ''
+  );
```

Comment:
Shifted this query into the because it wasn't used in the parent component now that data is forcibly refetched.

I wonder if there is some refactor option where the refetched data could be passed into this component?

This felt simple enough, so have left as is.

Reply:
Hm yeah agree, per above comment, I think `refetch` would have allowed the sharing. Happy with this as is, try not to change too much right before RC, but agree could be refactored